scripts/_draw_info_display.py

```diff
@@ -0,0 +1,226 @@
+from dolphin import event, gui, utils
+from Modules import mkw_classes as classes, mkw_core as core
+import configparser
+import math
+import os
+
+config = configparser.ConfigParser()
+
+config.read(os.path.join(utils.get_script_dir(), 'Modules', 'infodisplay.ini'))
+
+debug = config['DEBUG'].getboolean('Debug')
+
+frame_count = config['INFO DISPLAY'].getboolean('Frame Count')
+
+lap_splits = config['INFO DISPLAY'].getboolean('Lap Splits')
+
+speed = config['INFO DISPLAY'].getboolean('Speed')
+
+internal_velocity = config['INFO DISPLAY'
+                    ].getboolean('Internal Velocity (X, Y, Z)')
+internal_velocity_xyz = config['INFO DISPLAY'
+                        ].getboolean('Internal Velocity (XYZ)')
+
+external_velocity = config['INFO DISPLAY'
+                    ].getboolean('External Velocity (X, Y, Z)')
+external_velocity_xyz = config['INFO DISPLAY'
+                        ].getboolean('External Velocity (XYZ)')
+
+moving_road_velocity = config['INFO DISPLAY'
+                       ].getboolean('Moving Road Velocity (X, Y, Z)')
+moving_road_velocity_xyz = config['INFO DISPLAY'
+                           ].getboolean('Moving Road Velocity (XYZ)')
+
+moving_water_velocity = config['INFO DISPLAY'
+                        ].getboolean('Moving Water Velocity (X, Y, Z)')
+moving_water_velocity_xyz = config['INFO DISPLAY'
+                            ].getboolean('Moving Water Velocity (XYZ)')
+
+charges_and_boosts = config['INFO DISPLAY'
+                     ].getboolean('Charges and Boosts')
+checkpoints_and_completion = config['INFO DISPLAY'
+                             ].getboolean('Checkpoints and Completion')
+miscellaneous = config['INFO DISPLAY'
+                ].getboolean('Miscellaneous')
+
+surface_properties = config['INFO DISPLAY'
+                     ].getboolean('Surface Properties')
+
+position = config['INFO DISPLAY'].getboolean('Position')
+
+stick = config['INFO DISPLAY'].getboolean('Stick')
+
+color = int(config['INFO DISPLAY']['Text Color'], 16)
+
+digits = config['INFO DISPLAY'].getint('Digits (to round to)')
```

Comment:
We should probably limit this to a maximum of 7 digits. Anything beyond that is likely unnecessary.